modules/build/src/main/scala/scala/build/internal/AmmUtil.scala

```diff
@@ -18,5 +18,8 @@ object AmmUtil {
 
   def encodeScalaSourcePath(path: Seq[Name]) = path.map(_.backticked).mkString(".")
 
-  def normalizeNewlines(s: String) = s.replace("\r", "").replace("\n", System.lineSeparator())
+  def normalizeNewlines(s: String)(using newLine: String = lineSeparator): String =
```

Comment:
```suggestion
  def normalizeNewlines(s: String): String =
```
`newLine` seems to no longer be used here, no?